### ISSUE TITLE
XFA forms

### ISSUE THREAD

**Issue by rhousham** [Emano-Waldeck/pdf-reader#14]
A good xfa forms attempt - but seems to have some issues with the tables and adding/removing rows.

https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/866604/t132-eng.pdf

**Comment by Emano-Waldeck** [comment]
Right, PDF scripting is partially supported. I'll keep this open for now.
